charts/templates/statefulset.yaml

```diff
@@ -23,8 +24,104 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
-      {{- if ne .Values.bootstrapHostName "" }}
+      {{- if or ( ne .Values.bootstrapHostName "") (eq .Values.tls.enableOnDefaultAppServers true ) }}
       initContainers: 
+      {{- if eq .Values.tls.enableOnDefaultAppServers true }}
+      - name: copy-certs
+        image: "finalgene/openssh"
```

Comment:
Do we have the approval from Rangan and Legal Team to use this docker image?

Reply:
Will send email for legal team for approval.

Reply:
We should move to the official RedHat UBI image who has openssl in it.
Then we are sure that this third party docker image is well maintained and ensure that security vulnerabilities will be handled.

Reply:
I've push my change to redhat/ubi9:9.3. I've tried ubi8 but it seems that ubi8 does not have openssl, only ubi9 contains openssl.

Reply:
You're right RHEL UBI 8 does not have openssl.
I've checked and the minimal image do not have it.

You can recheck with legal to get approval for RHEL UBI9 usage then.